packages/secret-rotator/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use the official lightweight Node.js 12 image.
+# https://hub.docker.com/_/node
+FROM node:14-alpine
+
+# Create and change to the app directory.
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+# Install production dependencies.
+RUN npm install
```

Comment:
Take another look at the release-trigger Dockerfile. The way this is implemented (single stage), the final image will contain the node devDependencies which are not necessary at runtime.

Reply:
This can be done in a follow-up PR if desired.

Reply:
I think I addressed this... let me know if I missed something.